expo/expo
# fix(metro-config): Fix symbolication bug `ENOENT

# Why

- Alternative to disabling all frame customization https://github.com/expo/expo-cli/pull/4470
- This implements a hack to disable reading any invalid frame data that Metro may provide.
- Fixes ENG-5720


# How

- Set the lineNumber and column to null, this disables Metro's file reading because there's no relevant data.
- Also collapses the frame since it will mostly be useless.




# Test Plan

- Use @brentvatne's repro

- `npx expo customize` -> generate `metro.config.js`
- App.js
```tsx
import { Button, View } from 'react-native';

export default function App() {
  return (
    <View style={{ flex: 1, alignItems: "center", justifyContent: "center" }}>
      <Button
        title="Press me!"
        onPress={() => {
          console.warn("This is a console.warn");
          console.error("This is a console.error");
        }}
      />
    </View>
  );
}
```
- `npx expo start`
- press button, and see no errors in the terminal.



# Checklist
